src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Button,
+  createStyles,
+  Grid,
+  Switch,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+
+import { LayerType, TableType } from '../../../config/types';
+import { addLayer, removeLayer } from '../../../context/mapStateSlice';
+import { loadTable } from '../../../context/tableStateSlice';
+import { layersSelector } from '../../../context/mapStateSlice/selectors';
+import { menuList } from '../utils';
+
+function MenuItemMobile({ classes }: MenuItemMobileProps) {
+  const [expanded, setExpanded] = useState<string>('');
+
+  const selectedLayers = useSelector(layersSelector);
+
+  const dispatch = useDispatch();
+
+  const handleChange = (panel: string) => (
+    event: object,
+    newExpanded: boolean,
+  ) => {
+    setExpanded(newExpanded ? panel : '');
```

Comment:
why is expanded a string? Makes sense its a boolean...

Reply:
On second look, this makes sense. For the comment I mentioned [here](https://github.com/WFP-VAM/prism-frontend/pull/129/files#r593741103), it might be best to just name this file `MobileMenuItems` plural. What do you think?

Reply:
I tried to refactor the codes so that the approach is similar to `<MenuItem/>` component. I think the plural one should be the `const menu` since it consist multiple item. The `<MenuItem/>` or `<MenuItemMobile/>` itself is still singular. What do you think about that?

Reply:
It makes sense for `<MenuItem/>` to be singular since it's rendered multiple times with this array: 
https://github.com/WFP-VAM/prism-frontend/blob/22cc5e452ddf855965e800578c7dd0f55b36bff8/src/components/NavBar/index.tsx#L40

The new mobile component you made shouldn't be singular because you use the `menuList` inside of it, meaning we only render one `<MenuItemMobile/>`
https://github.com/WFP-VAM/prism-frontend/blob/011d9f95c397feca04992d4f984610ca6099ff6b/src/components/NavBar/MenuItemMobile/index.tsx#L54

Overall, the `MenuItemMobile` component should be refactored to be similar to `MenuItem`, or it should be renamed to something like `MenuItemsMobile` so it's name makes more sense :+1: